getting_started/setup/setup_linux_machine.rst

```diff
@@ -0,0 +1,142 @@
+.. _setup-linux-machine:
+
+Set up a Linux dev and test environment
+#######################################
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+************
+
+
+These instructions will guide you through the process of building the SOF branch of the Linux kernel from scratch and installing it on your machine. The kernel will be installed in addition to your default kernel, so if something goes wrong you can always revert to the default kernel.
```

Comment:
I think we should split this in two sections

a) compiling the kernel: this could be common with the ''Create a linux development environment" part
b) installing it with 
b1) on your local machine
b2) or a target with ktest.pl

Reply:
I'm not sure it's a great idea to combine them. The only part in common is the dependencies (which we could just link to on the ktest page). Ktest has you clone the upstream linux, then add sof as a remote whereas this page just has you clone the sof fork directly or download a zip to make it easier. After that, nothing more is shared between the different setups.

Reply:
We could directly clone SOF for ktest, that would make the explanations more consistent. In practice it's even better for SOF developers, where 'origin' becomes the SOF tree.

Reply:
I redid the organization of the getting started section and merged what I could from the two docs pages. Take a look and lmk if it's good.